### PR TITLE
Adding Quartz network

### PR DESCRIPTION
**Pull request by stepanLav**
# Basic data
Screenshot 2021-12-10 at 14 10 31

Url for check:
https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fquartz.unique.network#/rpc

### CODE REVIEW

File: chains/v2/types/quartz.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "runtime_id": 913010,
+  "types": {
+    "Balance": "u128",
+    "Index": "u64",
```
Comment (ERussel) [inline chains/v2/types/quartz.json:5]: I think it should be 32. Search first AccountInfo in metadata and then search by id of the nonce field's type

Comment (stepanLav) [reply chains/v2/types/quartz.json]: Ok, I was confused with AccountIndex, now i get it and fix

File: chains/v2/chains_dev.json [hunk 2]
```diff
@@ -898,5 +898,65 @@
         },
         "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Calamari.svg",
         "addressPrefix": 78
+    },
+    {
+        "chainId": "cd4d732201ebe5d6b014edda071c4203e16867305332301dc8d092044b28e554",
+        "parentId": "b0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafe",
+        "name": "Quartz",
```
Comment (antonkhvorov) [inline chains/v2/chains_dev.json:905]: QUARTZ
all capps

**Review by ERussel — APPROVED** [review]

File: chains/v2/chains_dev.json [hunk 3]
```diff
@@ -898,5 +898,65 @@
...
+            {
+                "assetId": 0,
+                "symbol": "QTZ",
+                "precision": 18
+            }
+        ],
+        "nodes": [
+            {
+                "url": "wss://quartz.unique.network",
+                "name": "Unic node"
```
Comment (antonkhvorov) [inline chains/v2/chains_dev.json:916]: Unique node

File: chains/v2/chains_dev.json [hunk 4]
```diff
@@ -898,5 +898,65 @@
...
+            },
+            {
+                "url": "wss://eu-ws-quartz.unique.network",
+                "name": "Unic europe node"
```
Comment (antonkhvorov) [inline chains/v2/chains_dev.json:920]: Unique Europe node

File: chains/v2/chains_dev.json [hunk 5]
```diff
@@ -898,5 +898,65 @@
...
+            },
+            {
+                "url": "wss://us-ws-quartz.unique.network",
```
Comment (antonkhvorov) [inline chains/v2/chains_dev.json:923]: Unique US node

**Pull request merged**